.gitmodules

```diff
@@ -55,3 +55,6 @@
 [submodule "gslib/vendored/oauth2client"]
 	path = gslib/vendored/oauth2client
 	url = https://github.com/gsutil-mirrors/oauth2client.git
+[submodule "third_party/google-auth-library-python"]
```

Comment:
You should also update https://github.com/GoogleCloudPlatform/gsutil/blob/6bfe364782ab3dc25c29606c421bc58e9ff7c6dc/gsutil.py#L71

Reply:
Done